shotgun_api3/lib/mockgun/mockgun.py

```diff
@@ -245,15 +244,17 @@ def schema_field_read(self, entity_type, field_name=None):
         else:
             return dict((k, v) for k, v in self._schema[entity_type].items() if k == field_name)
 
-
-    def find(self, entity_type, filters, fields=None, order=None, filter_operator=None, limit=0, retired_only=False, page=0):
-
+    def find(
+        self,
+        entity_type, filters, fields=None,
```

Comment:
this formatting seems a bit arbitrary. Any reason it's done this way?

Reply:
First line is self, next three is the most common-only used params to the method and then the rest. Should I simply break them on two lines? One arg per line seems wasteful to me.